### DISCUSSION TITLE
Version 7.3.1 out now

### DISCUSSION THREAD

**Discussion by guustnieuwenhuis** [MasaCMS/MasaCMS#62] [Announcements]
We've released a patch version with a couple of bug fixes:

- Improved extended attributes UI in the content manager
- Adobe ColdFusion compatibility fixes
- Improve customUI tab labels #56
- bug fixes for #41, #43, #44, #45, #51

**Comment by guustnieuwenhuis** [comment]
And a big thanks to @jimblesphere and @PeterBowater for there contributions!

**Comment by jimblesphere** [comment]
Happy to be able to contribute. I have done preliminary testing under both
CF 2016 and CF 2021 and have not found any incompatibilities.

jim
